### DISCUSSION TITLE
commentary plugin does not work in verison 1.8.0

### DISCUSSION THREAD

**Discussion by momadacoding** [JetBrains/ideavim#400] [Q&A]
[ideavimrc.zip](https://github.com/JetBrains/ideavim/files/7490832/ideavimrc.zip)
IDE info:
Rider for Unreal Engine 2021.2.1
Build #RDCPPP-212.5080.85, built on October 5, 2021
Licensed to liu hao
Subscription is active until March 31, 2022.
Runtime version: 11.0.11+9-b1577.1 amd64
VM: Dynamic Code Evolution 64-Bit Server VM by JetBrains s.r.o.
Windows 10 10.0
.NET Framework 4.0.30319.42000
GC: G1 Young Generation, G1 Old Generation
Memory: 1500M
Cores: 8
Registry: debugger.new.debug.tool.window.view=true, ide.tree.horizontal.default.autoscrolling=false, ide.tooltip.showAllSeverities=true, ide.mac.bigsur.alerts.enabled=TRUE, show.diff.preview.as.editor.tab.with.single.click=true, ea.enable.developers.list=false, parameter.info.max.visible.rows=10, ide.win.file.chooser.native=true, ide.mac.file.chooser.native=TRUE, search.everywhere.pattern.checking=false, performance.watcher.sampling.interval.ms=200, ide.borderless.tab.caption.in.title=false, awt.file.dialog.enable.filter=FALSE, debugger.show.values.use.inlays=TRUE, light.edit.file.open.enabled=false, performance.watcher.unresponsive.interval.ms=1000, vcs.enable.add.ignored.directories.to.exclude=false, ide.allow.merge.buttons=TRUE, search.everywhere.settings=true, use.winp.for.graceful.process.termination=FALSE, ide.tooltip.initialDelay=0, ide.require.transaction.for.model.changes=false, ide.debug.in.title=true, rdclient.asyncActions=false
Non-Bundled Plugins: com.microsoft.vso.idea (1.162.0), IdeaVIM (1.8.0), io.github.mishkun.ideavimsneak (1.2.0), unreal-link (2021.2.1.4)

**Comment by AlexPl292** [comment]
@lippfi could you please take a look?

**Comment by AlexPl292** [comment]
Please follow this issue: https://youtrack.jetbrains.com/issue/VIM-1758
